src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_BALL_NUMBER = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(random.nextInt(MAX_BALL_NUMBER));
+        ball.setColor(colorSupplier.getRandomColor());
+
+        return ball;
```

Comment:
We may introduce all-arguments constructor for ball
```suggestion
        return new Ball(random.nextInt(MAX_BALL_NUMBER), colorSupplier.getRandomColor());
```